components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -93,9 +102,27 @@ public UserStoreResponse addUserStore(UserStoreReq userStoreReq) {
 
         try {
             validateMandatoryProperties(userStoreReq);
+
+            String userstoreDomain = userStoreReq.getName();
+            String tenantDomain = ContextLoader.getTenantDomainFromContext();
+            List<LocalClaim> localClaimList = new ArrayList<>();
+
+            List<ClaimAttributeMapping> claimAttributeMappingList = userStoreReq.getClaimAttributeMappings();
+            if (claimAttributeMappingList != null) {
+                localClaimList =  createLocalClaimList(userstoreDomain,
+                        claimAttributeMappingList);
```

Comment:
Can we get this to a single line without exceeding char limit?

Reply:
Addressed with commit c731518fd0ea44d2d06c025d044e61114470